test/coding/test_default_policy_override.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from conftest import skip_docker
+
+from autogen.code_utils import is_docker_running
+from autogen.coding.docker_commandline_code_executor import DockerCommandLineCodeExecutor
+
+
+@pytest.mark.skipif(
+    skip_docker or not is_docker_running(), reason="Docker is not running or requested to skip Docker tests"
+)
+def test_policy_override():
```

Comment:
This is a great test, however, I think we should move this to `test/coding/test_commandline_code_executor.py` to be more consistent